backend/api/tests/test_models.py

```
@@ -0,0 +1,1415 @@
+            gewalttat=self.tat,
+            psychische_folgen="A",
+            koerperliche_folgen="N",
+            finanzielle_folgen="N",
+            arbeitseinschraenkung="N",
+            verlust_arbeitsstelle="N",
+            soziale_isolation="N",
+            suizidalitaet="N",
+            keine_angabe="N",
+        )
+
+        with self.assertRaises(Exception):
```

Ruff (0.14.10)

Do not assert blind exception: `Exception`

(B017)